tests/www/views/test_views_tasks.py

```diff
@@ -594,6 +622,38 @@ def _get_appbuilder_pk_string(model_view_cls, instance) -> str:
     return model_view_cls._serialize_pk_if_composite(model_view_cls, pk_value)
 
 
+@pytest.fixture(scope="function")
+def task_instance_to_delete(session):
+    dag = DagBag().get_dag("example_bash_operator")
+    task0 = dag.get_task("runme_0")
+    task0.task_id = "test_task_instance_to_delete"
+    execution_date = timezone.datetime(2021, 6, 26)
+    ti = TaskInstance(task0, execution_date, state="success")
```

Comment:
@Jorricks This is the cause of your failure now -- a recently merged change means that you can no longer create a TI in the database without a matching DagRun row.

Reply:
Thanks a lot for pinpointing that!!!! I will see if I can spend some time to fix the tests this week.